massa-node/base_config/bootstrap_whitelist.json

```diff
@@ -1,4 +1,2 @@
 [
```

Comment:
don't remember if have a empty array is a problem, maybe we should delete the file ?

Reply:
I think deleting it has a different meaning

Reply:
Indeed, deleting the file would be "I allow everyone to bootstrap on me".
Having an empty array means "I allow noone to bootstrap on me", which I think is the wanted behaviour?

Reply:
we could try to deploy before tagging in order to test

Reply:
empty whitelist == nobody can bootstrap. No whitelist, everyone can. Personally I think it's nice if we keep one of our IP in this file so that we will be able to ping nodes to have some kind of topology of the network

Reply:
> Indeed, deleting the file would be "I allow everyone to bootstrap on me". Having an empty array means "I allow noone to bootstrap on me", which I think is the wanted behaviour?

yes we want "nobody to bootstrap on me"

Reply:
I didnt had the full conversation when I wrote : https://github.com/massalabs/massa/pull/4612#discussion_r1442632919 but still relevant

Reply:
hmm see @AurelienFT 's comment below, it's also a good idea, assuming people would keep that port open and accessible

Reply:
> hmm see @AurelienFT 's comment below, it's also a good idea, assuming people would keep that port open and accessible

Not everyone but could help us in many cases in the future I think

Reply:
yeah good idea.
Maybe let's whitelist all of our bootstrap server IPs, both IPV4 and ipv6

Reply:
Done in a87836b484a7d12e8055e8ec0beeb5a1e7454d3a